core/src/test/java/org/infinispan/jmx/CacheMBeanTest.java

```diff
@@ -119,16 +121,6 @@ public void testDuplicateJmxDomainOnlyCacheExposesJmxStatistics() throws Excepti
       }
    }
 
-   public void testMalformedCacheName(Method m) throws Exception {
```

Comment:
Yeah I didn't mean the test itself was good.  But rather that we have a test that an exception is indeed thrown when we try to retrieve a cache that isn't defined.